Anabol hardcore

OPJerry opened the pull request:
Plik gestures.py to działający gest myszką. po zrobieniu kółka dookoła ekranu zostanie wykonane polecenie, w tym wypadku print(string)

OPJerry committed:
testowy_commit

OPJerry committed:
pierwsze realne postępy

OPJerry committed:
Gotowy dzialajacy gest myszka

OPJerry committed:
częściowo udane przyśpieszenie myski w pythonie

gimb commented on expect_us_anonymus.py:
```diff
@@ -0,0 +1,12 @@
+print('test')
```
to delete

gimb commented on faster_mouse_test.py:
```diff
@@ -0,0 +1,19 @@
+import pyautogui
```
it works :)

gimb commented on gestures.py:
```diff
@@ -0,0 +1,45 @@
+start_time = time.time()
+""" skrypt dowiaduje się która jest teraz godzina* """
+
+
```
this will be main function

gimb commented on gestures.py:
```diff
@@ -0,0 +1,45 @@
+    
+    if len(anserws) == 5:
+        print('wykonałeś gest - zrobiłeś prawdopodobnie kółko')
+        break
```
add
 if __name__ == "__main__":
    main()

gimb commented on gestures.py:
```diff
@@ -0,0 +1,45 @@
+import pyautogui
```
add docstrings

gimb commented on mouse_testing.py:
```diff
@@ -0,0 +1,38 @@
+from pynput.mouse import Button, Controller
```
docstring

OPJerry committed:
dodatkowe poprawki do kodu

OPJerry committed:
faster mouse code update

OPJerry commented:
I added new functionality to the code. Now, after running the script, the mouse will be accelerated for 30 seconds, after which the program will automatically turn off.

If the user makes a small movement of the mouse, the program will not run to still allow precise control

Commit:
Code review update

i changed some lines of code to make it clear

Commit:
new names

OPJerry committed:
czeski błąd.

OPJerry committed:
New system logic

adding complex code to check how much the user wants to speed up the mouse by running him through a series of 15-second tests. Then sets the speed increase of the mouse until the script is disabled

OPJerry committed:
deleted test1.txt

OPJerry committed:
Put on a helmet...

From today we fly without seat belts brothers

Commit:
backup

OPJerry committed:
update kodu

Commit:
last update befor pull request
